R/tm_data_table.R

```diff
@@ -12,6 +13,8 @@
 #' @param datasets_selected (`character`) A vector of datasets which should be
 #'   shown and in what order. Names in the vector have to correspond with datasets names.
 #'   If vector of length zero (default) then all datasets are shown.
+#'   Note: Only datasets of the `data.frame` class are compatible;
+#'   using other types will cause an error.
```

Comment:
```suggestion
#'   Note: Only datasets of the `data.frame` class are compatible; other data tupés will be skipped.

```

Not true. Other types will be skipped as there is a Filter call

Reply:
for `dataset_selected `if its not `data.frame` then it will fail assertion https://github.com/insightsengineering/teal.modules.general/blob/975d06400514b647d6f1f7c429b39611a5073b61/R/tm_data_table.R#L151-L155. other types are only skipped when set as` datanames `not `dataset_selected`

Reply:
Ok, fair enough

Reply:
Feel free to merge